Changelog.md

```diff
@@ -40,6 +42,9 @@
 - [#363]: Do not generate empty `Event::Text` events
 - [#412]: Fix using incorrect encoding if `read_to_end` family of methods or `read_text`
   method not found a corresponding end tag and reader has non-UTF-8 encoding
+- [#413]: `Attribute::from((&[u8], &[u8]))` was made consistent with `Attribute::from((&str, &str))`
+  in that both will now escape the value. WARNING: if you were passing escaped values via this
```

Comment:
I'm not sure how to draw more attention to this but it is a somewhat consequential change and users won't get a compile error.